apps/web/src/pages/workflow/components/import-modal/components/style.less

```diff
@@ -0,0 +1,64 @@
+.@{prefix}-import-box {
+    .@{prefix}-import-file {
+        display: flex;
+        flex: 1 0 0;
+        flex-direction: column;
+        gap: 10px;
+        align-items: center;
+        align-self: stretch;
+        justify-content: center;
+        padding: 35px 10px 41px;
+        background: var(--gray-1);
```

Comment:
Direct use of color variables should be avoided, and semantic variables can be used instead. In this scenario, `--component-background-gray` can be used instead.